docs/howto/auth-remote-user.txt

```diff
@@ -6,12 +6,11 @@ This document describes how to make use of external authentication sources
 (where the web server sets the ``REMOTE_USER`` environment variable) in your
 Django applications.  This type of authentication solution is typically seen on
 intranet sites, with single sign-on solutions such as IIS and Integrated
-Windows Authentication or Apache and `mod_authnz_ldap`_, `CAS`_, `Cosign`_,
+Windows Authentication or Apache and `mod_authnz_ldap`_, `CAS`_,
```

Comment:
Rewrap at 79 chars after `Cosign` removal, i.e.
```
Windows Authentication or Apache and `mod_authnz_ldap`_, `CAS`_, `WebAuth`_,
`mod_auth_sspi`_, etc.
```

Reply:
Okay. Fixing it now.